### PR TITLE
adding Geometry/CommonToplogies dependency to CUDADataFormats/TrackingRecHit

### PR DESCRIPTION
**Pull request by davidlange6**
in interface (newly added?) but missing from BuildFile.

### CODE REVIEW

File: CUDADataFormats/TrackingRecHit/BuildFile.xml [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <use name="CUDADataFormats/Common"/>
 <use name="CUDADataFormats/SiPixelCluster"/>
 <use name="DataFormats/Common"/>
+<use name="GeometryCommonTopologies"/>
```
Comment (fwyzard) [inline CUDADataFormats/TrackingRecHit/BuildFile.xml:6]: I guess it should be
```suggestion
<use name="Geometry/CommonTopologies"/>
```

But do we need an entry also for header-only dependencies ?

**Comment by makortel** [comment]
@cmsbuild, please test

**Comment by makortel** [comment]
+heterogeneous

**Comment by clacaputo** [comment]
+reconstruction

**Comment by qliphy** [comment]
+1

**Pull request merged**